### PR TITLE
ml-wrappers: Remove flake8 pins

### PR DESCRIPTION
**Pull request by gaugup**

### CODE REVIEW

File: python/ml_wrappers/dataset/dataset_wrapper.py [hunk 1]
```diff
@@ -45,11 +45,11 @@ def __init__(self, dataset, clear_references=False):
...
-                not isinstance(dataset, np.ndarray) and not issparse(dataset) and
-                not str(type(dataset)).endswith(".DenseData'>") and
-                not str(type(dataset)).endswith("torch.Tensor'>") and
-                not str(type(dataset)).endswith("BatchDataset'>")):
+        if not isinstance(dataset, pd.DataFrame) and not isinstance(dataset, pd.Series) and \
+                not isinstance(dataset, np.ndarray) and not issparse(dataset) and \
+                not str(type(dataset)).endswith(".DenseData'>") and \
+                not str(type(dataset)).endswith("torch.Tensor'>") and \
+                not str(type(dataset)).endswith("BatchDataset'>"):
             raise TypeError("Got type {0} which is not not supported in DatasetWrapper".format(
```
Comment (imatiach-msft) [inline python/ml_wrappers/dataset/dataset_wrapper.py:53]: nit: fix type error message here
not not supported -> not supported

File: tests/main/test_timestamp_featurizer.py [hunk 2]
```diff
@@ -23,9 +23,9 @@ def test_no_timestamps(self, iris):
         featurizer = CustomTimestampFeaturizer(iris[DatasetConstants.FEATURES]).fit(x_train)
         result = featurizer.transform(x_test)
         # Assert result is same as before, pandas dataframe
-        assert(isinstance(result, pd.DataFrame))
+        assert (isinstance(result, pd.DataFrame))
```
Comment (imatiach-msft) [inline tests/main/test_timestamp_featurizer.py:26]: nit: I think we can remove quotes here as well on these asserts

**Review by imatiach-msft — APPROVED** [review]

File: tests/main/test_timestamp_featurizer.py (hunk 2 above)
Comment (gaugup) [reply tests/main/test_timestamp_featurizer.py]: done

File: python/ml_wrappers/dataset/dataset_wrapper.py (hunk 1 above)
Comment (gaugup) [reply python/ml_wrappers/dataset/dataset_wrapper.py]: done

**Pull request merged**